src/components/CallToAction/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent } from 'react'
+import { Container, Row, Col } from 'reactstrap'
+import cn from 'classnames'
+
+import { SafeExternalLink } from '../Link'
+import { blurryBackgroundBanner } from '../../scss/global.module.scss'
+
+export interface CallToActionProps {
+  headline: string
+  description: string
+  Link?: any
+  buttonLabel: string
+  url: string
+}
+
+const CallToAction: FunctionComponent<CallToActionProps> = ({
+  headline,
+  description,
+  Link = SafeExternalLink,
+  buttonLabel,
+  url,
+}) => (
+  <div className={cn(blurryBackgroundBanner, 'text-center', 'py-md-5')}>
```

Comment:
Why is the `blurryBackgroundBanner` set here too? Isn't it already part of the Hero section?

Reply:
It is being used in both places. That's why I extracted it to a global css-module.